drakrun/drakrun/draksetup.py

```diff
@@ -245,6 +245,53 @@ def send_usage_report(report):
         logging.exception("Failed to send usage report. This is not a serious problem.")
 
 
+def create_rekall_profiles(install_info, pdb_guid):
+    profiles_path = os.path.join(LIB_DIR, "profiles")
+
+    with tempfile.TemporaryDirectory() as mount_path:
+        # we mount 2nd partition, as 1st partition is windows boot related and 2nd partition is C:\\
+
+        if install_info["storage_backend"] == "zfs":
+            # workaround for not being able to mount a snapshot
+            base_snap = shlex.quote(os.path.join(install_info["zfs_tank_name"], 'vm-0@booted'))
+            tmp_snap = shlex.quote(os.path.join(install_info["zfs_tank_name"], 'tmp'))
+            subprocess.check_output(f'zfs clone {base_snap} {tmp_snap}', shell=True)
+
+            tmp_mount = shlex.quote(os.path.join("/dev/zvol", install_info["zfs_tank_name"], "tmp-part2"))
+            subprocess.check_output(f'mount -t ntfs -o ro {tmp_mount} {mount_path}')
+        else:  # qcow2
+            subprocess.check_output("modprobe nbd")
+            # TODO: this assumes /dev/nbd0 is free
+            subprocess.check_output(' '.join([
```

Comment:
AFAIK, `' '.join(` is unnecessary here.
We can just call `subprocess.check_output(["cmd", "arg1", "arg2"])`

Then, we can also drop `shell=True`

Reply:
This has been copied from https://github.com/CERT-Polska/drakvuf-sandbox/blob/master/drakrun/drakrun/draksetup.py#L155-L162

Reply:
Care to explain why we can drop `shell=True`?

Reply:
well, shell is only required when we need shell-like abilities, for example parsing a command like `touch a b c`  or `cat xd > xd2` or`modprobe nbd` (so please fix it)
it will try to run a binary named `modprobe nbd` which obviously doesn't exist

but if we're providing a list of arguments, we don't need shell to split whole command to split into arguments.

the code mentioned isn't wrong in the sense that it's not working, but it's somewhat useless

Reply:
just try running 
` subprocess.check_output("ls /")` -> errors
` subprocess.check_output("ls /", shell=True)` -> ok